src/components/common/FloatingVariablesSuggestions/Suggestions.tsx

```diff
@@ -164,7 +164,7 @@ const Suggestions = ({ handleDeActivation, loading, variables, reloadVariables,
         if (variables?.length === 0) {
             return (
                 <>
-                    <div className="flexbox-col dc__align-self-stretch dc__overflow-scroll flex-grow-1 h-200">
+                    <div className="flexbox-col dc__align-self-stretch dc__overflow-auto flex-grow-1 h-250">
```

Comment:
Why have we increased the height. if it is from Product, it should be same on the loading as well

Reply:
With 200 height content was overflowing in empty state. So after discussion with Product have increased the height to avoid unnecessary scroll in empty state